urdf/inc/ur_common.xacro

```diff
@@ -21,8 +21,29 @@
     </inertial>
   </xacro:macro>
 
+  <xacro:macro name="get_visual_params" params="name:=^ type:=^" >
```

Comment:
This is great!

Reply:
Maybe we could restrict this PR to this feature? I also agree, that this is a really nice addition, the link generation as discussed in https://github.com/UniversalRobots/Universal_Robots_ROS2_Description/pull/17/files#r797535820 raises more discussion points.

Reply:
Oki. I did the "get_visual_params" part the quickest, as it was the only way to get the get_mesh done. I was thinking about another refactoring around the "get_visual_params", but that would also hide more information.. So I guess I won't do that for now at least.